manager/src/grype_db_manager/grypedb.py

```diff
@@ -462,11 +462,9 @@ def run(self, *args, provider_root_dir: str, config: str) -> int:
 
         env = dict(  # noqa: PIE804
             **os.environ.copy(),
-            **{
-                "GRYPE_DB_VUNNEL_ROOT": provider_root_dir,
-                "GRYPE_DB_CONFIG": config,
-                "GRYPE_DB_LOG_LEVEL": level,
-            },
+            GRYPE_DB_VUNNEL_ROOT=provider_root_dir,
```

Comment:
I assume this is an unrelated change just making the Python nicer?

Reply:
ruff did this -- I like it